src/QMCApp/QMCMain.cpp

```diff
@@ -66,8 +66,8 @@ QMCMain::QMCMain(Communicate* c)
       traces_xml(NULL)
 #endif
 {
-  Communicate node_comm;
-  node_comm.initializeAsNodeComm(*OHMMS::Controller);
+  Communicate node_comm{OHMMS::Controller->NodeComm()};
+// node_comm.initializeAsNodeComm(*OHMMS::Controller);
```

Comment:
Commented-out code

Reply:
good catch, I don't know why the tests didn't catch this. Should I push the change or is it already applied to the PR?

Reply:
Push the change.
We don't have any tools or tests to catch commented-out code. 
(I do this all the time, too - comment out code temporarily while developing, and sometimes miss it when checking  in the code)

Reply:
Ah, sorry I was confused (the code reviews in github) always trip me. *I* did the change (and forgot to remove the comment), which I didn't remember doing but now I do. 

Maybe a way to avoid commented code is to 1) use .clang-tidy or .clang-format or cppcheck to complain about badly formatted comment (they can do it) and 2) get used to badly format commented code on purpose during development (for example not leaving a space after `//`).